<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Equation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="equationEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="evalButton">
       <property name="text">
        <string>Evaluate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="plotCheckbox">
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CoordinatePlotWidget" name="plot"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoordinatePlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>uiModules.CoordinatePlotWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
